src/extension/module.js

```diff
@@ -2337,7 +2339,20 @@ import sampleRUM from './rum.js';
             custom: true,
             id: id || `custom-plugin-${i}`,
             condition,
-            button: {
+            container: containerId,
+          };
+
+          if (isBadge) {
+            plugin.feature = true;
+            plugin.elements = [{
+              tag: 'span',
+              text: title,
+              attrs: {
+                class: `hlx-sk-badge hlx-sk-badge-${badgeVariant?.toLowerCase() || 'default'}`,
```

Comment:
Wondering if we should check the value of `badgeVariant` against a list of allowed values and use `default` if it doesn't macth anything?